package/firmware/murata-firmware/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2024 Bootlin
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=murata-firmware
+PKG_VERSION:=2020.09.14
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/murata-wireless/cyw-fmac-fw.git
+PKG_MIRROR_HASH:=c2fc14802860f923d7bea36a709efeeffce4637a57084d78133e080b3dd8a87c
+PKG_SOURCE_VERSION:=52174a18134c7ef4a674ecd9fb68fc6e2bced969
+
+PKG_MAINTAINER:=Thomas Richard <thomas.richard@bootlin.com>
+PKG_LICENSE_FILES:=LICENCE
```

Comment:
The license looks ok:
> Subject to the terms and conditions of this
Agreement, Cypress Semiconductor Corporation ("Cypress") grants you a
non-exclusive, non-transferable license under its copyright rights in the
Software to reproduce and distribute the Software in object code form only,
solely for use in connection with Cypress integrated circuit products
("Purpose").

The firmware situation around the cypress chips is a mess. It would be so much easier if this would be in linux-firmware. 
@thom24 This is not your fault and you can probably also not improve this much.

Reply:
I used firmware and configs provided by ST.
It seems murata provides cypress firmware with custom CLM Blob.
There are cypress firmware and CLM Blob in linux-firmware.
But they are older than in cypress repository.

I don't really know how to deal with all of this.
Is it ok for you @hauke to use firmware, CLM blob and config from murata repositories ?
If not what do you expect ?

Reply:
yes using the firmware blobs from murata is fine.